### PR TITLE
Resolves #3771 - Seperate variables for calendar.php

### LINKED ISSUE #3771: add another variable with a unique style
**calendar**
template: calendar_weekrow_day

There is a variable ({$day_class}) which shows 3 styles which are (.trow1, .trow2 and .trow_sep), I ask for a unique variable for the style .trow_sep, which indicates the current day we are on the calendar EXAMPLE:

image

Here's the problem (at least for me, I don't know if there is another solution), I don't want to use the trow1 and trwo2 styles for "x" reason and I just want to use the trow_sep style. But it won't be possible because it will always give me all three styles

Will there be another solution to what I ask? regards

### PR DESCRIPTION
**Pull request by Ben-MyBB**
Marking this as a draft for now until I have had a look at the upgrade script and throughly tested this, but I wanted to open this for feedback on my method. An inital start on resolving #3771. Upgrade script still needs to be considered before merging to insert the additional two templates required.

All feedback is welcomed since I am still learning PHP :)

Resolves #3771

### CODE REVIEW

**Comment by Ben-MyBB** [comment]
Note: I have set the template versions to 1823 but I will need to update them again after the release of .23 if this PR hasn't been merged.

**Comment by Ben-MyBB** [comment]
@Eldenroot can you review?

File: calendar.php [hunk 1]
```diff
@@ -2548,19 +2548,21 @@
 			// Is the current day
 			if($day.$calendar_month.$year == $today && $month == $calendar_month)
 			{
-				$day_class = "trow_sep";
+				$current_day = "trow_sep";
```
Comment (euantorano) [inline calendar.php:2551]: With these separate templates, I wonder if we can get rid of these hard-coded CSS classes in the PHP code and just put the classes in the templates?

Comment (Ben-MyBB) [reply calendar.php:2551]: I can take a look but I want to get a fix for #3753 first since that's an important one 👍

**Comment by Ben-MyBB** [comment]
@euantorano updated :)

**Comment by euantorano** [comment]
Looks good to me, just need to test.

**Comment by Eldenroot** [comment]
Should be fine, personally I dont use calendar in MyBB, so just tested briefly.

**Pull request merged**